templates/core/about/builds.html

```diff
@@ -7,72 +7,75 @@
 	<h1>Builds</h1>
 	<div class="container about">
 	<p>
-		Docs.rs automatically builds crates' documentation released on
-		<a href="https://crates.io/">crates.io</a>
-		using the nightly release of the Rust compiler.
-		Builds can take a while depending how many crates are in <a href="/releases/queue">the queue</a>.
+		Docs.rs automatically builds documentation for crates released on <a href="https://crates.io/">crates.io</a>.
+		 
+		It may take a while to build your crate depending on how many crates are in <a href="/releases/queue">the queue</a>.
 	</p>
 
 	<p>
+		All crates are build in a sandbox using the nightly release of the Rust compiler.
 		{%- if rustc_version %}
-		The current version of the Rust compiler in use is <code>{{ rustc_version }}</code>.
+		The current version in use is <code>{{ rustc_version }}</code>.
 		{%- endif -%}
 	</p>
 
-	<p>
-		The README of a crate is taken from the <code>readme</code> field defined in
-		<code>Cargo.toml</code>. If a crate doesn't have this field,
-		no README will be displayed.
-	</p>
+	<h3>Notes on using Docs.rs</h3>
 
-	<h3>Diagnosing a failed build</h3>
-
-	<h4>Missing dependencies</h4>
+	<h4>Setting a README</h4>
 	<p>
-		Missing dependencies are a common reason for a failed build.
-		Docs.rs dependencies are managed through
-		<a href="https://github.com/rust-lang/crates-build-env">crates-build-env</a>;
-		see <a href="https://forge.rust-lang.org/docs-rs/add-dependencies.html">Forge</a> for instructions
-		on how to make a PR. You can always <a href="{{ docsrs_repo | safe }}/issues">file an issue</a>
-		if you're having trouble.
+		The README of a crate is taken from the <code>readme</code> field defined in
+		<code>Cargo.toml</code>. Omitting this field means that no README will be displayed.
 	</p>
 
 	<h4>Detecting Docs.rs from <code>build.rs</code></h4>
 	<p>
-		Docs.rs builds crates with the environment variable <code>DOCS_RS</code> set to
-		<code>1</code>, which enables the crate to detect docs.rs and build the
-		crate differently. This can be helpful if you need
+		Docs.rs sets the environment variable <code>DOCS_RS</code> to <code>1</code>.
+		Detecting Doc.rs and building your crate differently can be helpful if you need
 		dependencies for building the library, but not for building the documentation.
 	</p>
 
 	<h4>Detecting Docs.rs from <code>#[cfg]</code> attributes</h4>
 	<p>
-		You can detect Docs.rs by having a <a href="https://doc.rust-lang.org/cargo/reference/features.html">feature</a>
-		which is only set by Docs.rs. See <a href="metadata">Metadata</a> for more information.
+		Docs.rs sets a <a href="https://doc.rust-lang.org/cargo/reference/features.html">feature</a>. See <a href="metadata">Metadata</a> for details.
```

Comment:
I think we should add a better snippet giving an example though, like
```toml
[package.metadata.docs.rs]
rustc-args = ["--cfg", "docsrs"]
```
for `#[cfg(docsrs)]`

Reply:
cfg isn't a great example though because you already have `cfg(doc)`. I would rather give the example `all-features = true`.

Reply:
Now that I think about it, should probably have a "common metadata" section in the Metadata tab that gives common/useful configurations (separate PR though)

Reply:
`cfg(doc)` is very different than `cfg(docsrs)` because docs.rs runs nightly, so I think it's a very useful example to allow docs to start testing nightly features before they're stabilized.

Reply:
I've rewritten the section in hope for more clarity and added an example (`all-features = true`).

Reply:
Could you use the example I have above specifically?

Reply:
Done